boards/posix/native_posix/doc/index.rst

```diff
@@ -351,7 +351,7 @@ side-effects.
 Architecture
 ************
 
-.. figure:: native_layers.svg
+.. figure:: native_layers.jpg
```

Comment:
You don't want to convert vector graphic files. (they tend to be much smaller, they look better, they are editable and may be the source itself for the drawing)

Reply:
will fix that, it's an issue in the script thanks for spotting this one